ranking/get_cached_top_x_ranking.go

```diff
@@ -0,0 +1,76 @@
+package ranking
+
+import (
+	"github.com/PretendoNetwork/nex-go"
+	ranking "github.com/PretendoNetwork/nex-protocols-go/ranking"
+	ranking_types "github.com/PretendoNetwork/nex-protocols-go/ranking/types"
+)
+
+func getCachedTopXRanking(err error, client *nex.Client, callID uint32, category uint32, orderParam *ranking_types.RankingOrderParam) uint32 {
+	if commonRankingProtocol.getRankingsAndCountByCategoryAndRankingOrderParamHandler == nil {
+		logger.Warning("Ranking::GetCachedTopXRanking missing GetRankingsAndCountByCategoryAndRankingOrderParamHandler!")
+		return nex.Errors.Core.NotImplemented
+	}
+	rmcResponse := nex.NewRMCResponse(ranking.ProtocolID, callID)
+	server := client.Server()
+
+	if err != nil {
+		logger.Error(err.Error())
+		rmcResponse.SetError(nex.Errors.Ranking.Unknown)
```

Comment:
Change all of these with returns please. This also reduces nesting for error checks.
Also, can you change the error returned from the first error check to `Ranking.InvalidArgument`?

```suggestion
		return nex.Errors.Ranking.InvalidArgument
```